Plugin/Source/GUI/SettingsButton.cpp

```diff
@@ -10,7 +10,7 @@ SettingsButton::SettingsButton (const ChowtapeModelAudioProcessor& processor, ch
                                                                                                                   proc (processor),
                                                                                                                   openGLHelper (oglHelper)
 {
-    pluginSettings->addProperties ({ { openglID, false } }, this);
+    pluginSettings->addProperties<&SettingsButton::globalSettingChanged> ({ { openglID, false } }, *this);
```

Comment:
I pretty much copied this logic from BYOD, let me know if there's anything that should be different for ChowTape